XiEditor/XiTextPlane/TextPlane.swift

```diff
@@ -116,10 +118,29 @@ class TextPlaneLayer : NSOpenGLLayer {
             0
         ]
         return NSOpenGLPixelFormat(attributes: attr)!.cglPixelFormatObj!
-        
+    }
+
+    func changed(fps: Double) {
+        // print("Fps \(fps), ms/frame = \(1000.0 / fps)")
+    }
+
+    func changed(fpsStats stats: FpsSnapshot) {
+        print("Fps mean: \(stats.meanFps()), 99%: \(stats.fps(percentile: 0.01)), min: \(stats.minFps()), max: \(stats.maxFps())")
```

Comment:
maybe we also want to put this behind an ifdef?

Reply:
I'll hide it behind a debug "show fps" option.

Reply:
Elsewhere you'd used an `#if DEBUG` to gate a print statement, I thought that was a good idea, and seems equally suited here. Your idea makes sense too, but it might be confusing to actual users, since it won't show anything unless you run from xcode (at least until we wire up an overlay)

Reply:
I think #if DEBUG is too annoying since most developers won't care about FPS most of the time & it's pretty spammy.  Besides a UI label is nicer than console prints most of the time.